galeracluster/source/documentation/scriptable-sst.rst

```diff
@@ -103,6 +103,12 @@ These parameters are passed to all state transfer scripts, regardless of method
 
    ``--defaults-file`` The script is given the path to the ``my.cnf`` configuration file.
 
+   ``--total N`` This progress reporting parameter indicates the new SST stage and reports the estimated total work.
```

Comment:
This is not an SST script parameter, just like "complete". This is optional script output. For example, the script output on DONOR over the period of SST may look like:
```
flush tables
total 534
continue
complete 122
complete 340
complete 610
total 23
complete 20
done
```
Here we had two phases, one estimated 543 units of work length, another 23. First phase estimation was too small and we ended up doing a bit more. Second phase estimation may have been correct, but it was over before the full progress could be reported. Both `total` and `complete` messages are optional. In  the absence of those, we conclude that progress cannot be evaluated and just wait for `done` that unequivocally indicates that SST is finished.

The script outputs those control messages to standard output from where they are read by the parent mysqld process.